Sming/Makefile-project.mk

```diff
@@ -205,6 +205,9 @@ endif
 
 # libraries used in this project, mainly provided by the SDK
 LIBS		= microc microgcc hal phy pp net80211 $(LIBLWIP) wpa $(LIBMAIN) $(LIBSMING) crypto $(LIBPWM) smartconfig $(EXTRA_LIBS)
+ifeq ($(ENABLE_WPS),1)
```

Comment:
OK, i have add the switches in the same kind as i do it in Makefile-project. Because i don't use rboot, i have no chance to test it. (I hoppe i have add all correct).